src/app/onion/onion.routing.ts

```diff
@@ -15,19 +13,12 @@ import { OldDashboardComponent } from './old-dashboard/old-dashboard.component';
  *
  * @author Sean Donnelly
  */
-const dashboard = environment.experimental
-  ? {
+const dashboard =  {
```

Comment:
Now that we're not conditionally routing to different dashboards, can we move this back into the onion_routes object?

Reply:
I don did it in 6d301f5